elections/steering/2022/templates/election-templates.md

```diff
@@ -56,3 +56,110 @@ Noah Kantrowitz
 Kaslin Fields
 Davanum Srinivas
 
+== Candidate Qualification
+
+Per the requirements of the https://git.k8s.io/steering/elections.md[candidate process], I have verified that {name of candidate} has the necessary endorsements to qualify to run for steering committee.
+
+{candidate}, please prepare a bio for yourself based on the https://github.com/kubernetes/community/blob/master/elections/steering/2022/nomination-template.md[nomination-template.md] file and submit a PR before September 2nd.
+
+== Reminder 1 week before the end of nominations:
+
+Kubernetes Devs:
+
+Election update!
+
+We have XXX candidates[1] who have been nominated to run for Steering Committee, which is already a lot! But you have until August 30th to nominate additional candidates, using the nomination process[2].
+
+Contributors, now is also the time to check your eligibility by visiting the elections app[3] and seeing if you are marked Eligible. If you are not and should be, you can request an exception[4] in that app.
```

Comment:
"check your eligibility"... I assume this means "eligibility to vote", but since we were just suggesting that people nominate candidates and perhaps even suggest themselves as candidates, someone might be unsure which eligibility you're talking about. Perhaps "check your eligibility as voters" or "check your eligibility to vote" would be warranted.

Reply:
thanks. rephrased as `check your eligibility to vote`